pkg/graph/emulator/emulator.go

```diff
@@ -70,6 +71,7 @@ func (e *Emulator) GenerateDummyCR(gvk schema.GroupVersionKind, schema *spec.Sch
 	cr.SetKind(gvk.Kind)
 	cr.SetName(fmt.Sprintf("%s-sample", strings.ToLower(gvk.Kind)))
 	cr.SetNamespace("default")
+	cr.SetResourceVersion(fmt.Sprintf("%d", e.rand.Intn(1000)))
```

Comment:
nice! we already allow CEL references to all metadata fields so this makes sense to me